app/controllers/IndexController.scala

```diff
@@ -49,8 +51,9 @@ class IndexController @Inject()(
   ec: ExecutionContext)
     extends TaxsController(mcc, genericErrorView, tokenErrorView) {
 
-  def authorisedIndex: Action[AnyContent] = authAction.async { request: AuthenticatedRequest[_] =>
-    agentAwareShow(request)
+  def authorisedIndex: Action[AnyContent] = (authJourney.authWithSelfAssessment).async {
```

Comment:
Do we need the `authJourney.authWithSelfAssessment` action on all SA routes? Just wondering what might happen if someone bookmarks a link or is getting directly linked into mid service